src/api/readCpmProduct/src/v1/steps.py

```diff
@@ -0,0 +1,22 @@
+from aws_lambda_powertools.utilities.data_classes import APIGatewayProxyEvent
+from domain.core.cpm_product.v1 import CpmProduct
+from domain.repository.cpm_product_repository.v3 import CpmProductRepository
+from event.step_chain import StepChain
+
+
+def read_cpm_product(data, cache) -> CpmProduct:
```

Comment:
Do we want to split out into multiple steps? I'm asking not telling! Ha! 
We have done that for other lambdas, e.g. parse_query_params, parse_incoming_json etc.
In the search I have a step called
def parse_incoming_path_parameters(data, cache) -> str:

Reply:
Ah think I followed the read_device pattern, that just had the one function. 

I'm happy to split it out, keeps the functions simpler and easy to read/follow too